packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -79,11 +109,76 @@ class WorkerPool(SyftObject):
 
     __attr_unique__ = ["name"]
     __attr_searchable__ = ["name", "syft_worker_image_id"]
+    __repr_attrs__ = [
+        "name",
+        "syft_worker_image_id",
+        "max_count",
+        "workers",
+        "created_at",
+    ]
 
     name: str
     syft_worker_image_id: UID
+    syft_worker_image_name_tag: Optional[str]
```

Comment:
This should just be `syft_worker_image_tag`. I also think just `worker_image_id` and `worker_image_tag` is better, or even just `image_id` and `image_tag`. @rasswanth-s what do you think?

Reply:
@kiendang  image_id and image_tag is more better 👍

Reply:
Now pass the whole `SyftWorkerImage` 
![image](https://github.com/OpenMined/PySyft/assets/88959106/2287be7a-e026-4993-bee3-f03eac8595d4)